sensor/common/clusterentities/store_test.go

```diff
@@ -15,272 +14,88 @@ type ClusterEntitiesStoreTestSuite struct {
-					}
-				}
-				entityStore.RecordTick()
-			}
-		})
-	}
-}
+			s.Len(entityStore.podIPsStore.historicalIPs, tc.wantHistorySizeOnline)
+			s.Len(entityStore.endpointsStore.reverseHistoricalEndpoints, tc.wantHistorySizeOnline)
+			s.Len(entityStore.containerIDsStore.historicalContainerIDs, tc.wantHistorySizeOnline)
 
-func (s *ClusterEntitiesStoreTestSuite) TestChangingIPsAndExternalEntities() {
-	entityStore := NewStore()
-	type eUpdate struct {
-		containerID string
-		ipAddr      string
-		port        uint16
-		incremental bool
-	}
-	cases := map[string]struct {
-		entityUpdates     []eUpdate
-		expectedEndpoints []string
-	}{
-		"Incremental updates to the store shall not loose data": {
-			entityUpdates: []eUpdate{
-				{
-					containerID: "pod1",
-					ipAddr:      "10.0.0.1",
-					port:        80,
-					incremental: true,
-				},
-				{
-					containerID: "pod1",
-					ipAddr:      "10.3.0.1",
-					port:        80,
-					incremental: true,
-				},
-			},
-			expectedEndpoints: []string{"10.0.0.1", "10.3.0.1"},
-		},
-		"Non-incremental updates to the store shall overwrite all data for a key": {
-			entityUpdates: []eUpdate{
-				{
-					containerID: "pod1",
-					ipAddr:      "10.0.0.1",
-					port:        80,
-					incremental: true,
-				},
-				{
-					containerID: "pod1",
-					ipAddr:      "10.3.0.1",
-					port:        80,
-					incremental: false,
-				},
-				{
-					containerID: "pod2",
-					ipAddr:      "10.0.0.2",
-					port:        80,
-					incremental: true,
-				},
-			},
-			expectedEndpoints: []string{"10.3.0.1", "10.0.0.2"},
-		},
-	}
-	for name, tCase := range cases {
-		s.Run(name, func() {
-			for _, update := range tCase.entityUpdates {
-				entityStore.Apply(map[string]*EntityData{
-					update.containerID: entityUpdate(update.ipAddr, update.port),
-				}, update.incremental)
-			}
-			for _, expectedEndpoint := range tCase.expectedEndpoints {
-				result := entityStore.LookupByEndpoint(buildEndpoint(expectedEndpoint))
-				s.Lenf(result, 1, "Expected endpoint %q not found", expectedEndpoint)
-			}
+			s.T().Logf("%s", string(entityStore.Debug()))
+
+			// Transition to offline
+			entityStore.Cleanup()
+
+			s.Len(entityStore.podIPsStore.ipMap, tc.wantMapSizeOffline, "error in current IPs after cleanup")
+			s.Len(entityStore.endpointsStore.endpointMap, tc.wantMapSizeOffline, "error in current endpoints after cleanup")
+			s.Len(entityStore.containerIDsStore.containerIDMap, tc.wantMapSizeOffline, "error in current container IDs after cleanup")
+
+			s.Len(entityStore.podIPsStore.historicalIPs, tc.wantHistorySizeOffline, "error in historical IPs after cleanup")
+			s.Len(entityStore.endpointsStore.historicalEndpoints, tc.wantHistorySizeOffline, "error in historical endpoints after cleanup")
+			s.Len(entityStore.containerIDsStore.historicalContainerIDs, tc.wantHistorySizeOffline, "error in historical container IDs after cleanup")
 		})
 	}
 }
 
-// endregion
+// region external-entities test
```

Comment:
[nit]
```suggestion
```

Reply:
Yeah, this file used to be super-long and it had regions. Now they are gone and this line shall be removed.